src/NServiceBus.Core/MessageInterfaces/MessageMapper/Reflection/MessageMapper.cs

```diff
@@ -248,6 +248,15 @@ static string GetTypeName(Type t)
             return t.FullName;
         }
 
+        internal void EnsureTypesAreInitialized(List<Type> messageTypes)
```

Comment:
this looks more like a `TryInitialize` method and I wonder whether we wouldn't use that everywhere anyway?